public/js/monitor.js

```diff
@@ -212,6 +233,16 @@ function handleFormSubmits(formEvent) {
     setModalTabbing();
     return;
   }
+
+  if (formEvent.target.id === "unsubscribe-form") {
```

Comment:
This code won't be reached because there's already a check for this above (`if (formEvent.target.id === "unsubscribe-form") {`) which just calls `return`?